solver.py

```diff
@@ -279,6 +325,9 @@ def get_solution(self):
         cur_node = self.root
         for _ in range(self.target_set_size):
             cur_node = cur_node.best_choice(c=0.0)
+            # This has the algorithm select the next best action by picking the highest
+                # _average_ value node ('max child' choice). Another common option is 'robust child', which is picking the one with
+                # the highest visit count. I wonder if there would be performance changes b/w the two.
```

Comment:
Nevermind, I now understand why max child is taken. This is after the tree is fully expanded, so we are directly interested in the _best_ choice.